me.ppvan.psequel.json

```diff
@@ -38,8 +38,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/ppvan/psequel",
-                    "tag": "v0.1.4",
-                    "commit": "63c9afc4fd08ad37ec16ca34009f2df31894b4d4"
+                    "branch": "main",
```

Comment:
NEVER A BRANCH. This will break next time there is a commit.

Reply:
ok maybe i 'll create  a tag then, is there others way instead a new tag for each changes?

Reply:
simply a commit works.